### ISSUE TITLE
The github extension cannot show up on Edge browser

### ISSUE THREAD

**Issue by ruixinxu** [jupyterlab/jupyterlab-github#68]
The github extension cannot show up on Microsoft Edge browser (but it works fine on Chrome).

**Comment by ian-r-rose** [comment]
Hi @ruixinxu, it looks like MS Edge doesn't support the `TextEncoder` API, which is used by this extension: https://caniuse.com/#feat=textencoder .
We could try to incorporate a polyfill, but it would be nice if MS could support more modern web APIs 😄

**Comment by heart5** [comment]
I know it now. Edge is evil.

**Comment by bollwyvl** [comment]
Non-chromium edge is no longer supported, and probably difficult to even install/keep installed, so this can likely be closed.
